### PR TITLE
Final milestone delivery

### PR DESCRIPTION
**Pull request by sergiogval**
## Create dynamical pages. 

This branch includes a code that creates cards and modals dynamically using JS. It complies with the milestone requirements as follow. 

- [x]     When the user clicks (or taps) the button to check project details, the popup with details about the project appears.

- [x]     When the user clicks (or taps) the close (X) button, the popup disappears.

- [x]      You should implement popups for both mobile and desktop screen sizes

 - [x]    You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma.

- [x]     In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.

 
- [x]        For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
  

- [x]       You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.

        After that, you could implement the popup window.

- [x]     You need to use pair-programming for this project. We will check to make sure that both partners have contributed to the project by looking at the commit history for each project. Therefore you need to implement the above features in all of the portfolios in your pair-programming group.

### CODE REVIEW

File: js/dinamic.js [hunk 1]
```diff
@@ -1,54 +1,107 @@
...
+    modalTitle.innerText = projects[i].title;
+    languageOne.innerText = projects[i].languages[0];
+    languageTwo.innerText = projects[i].languages[1];
+    languageThree.innerText = projects[i].languages[2];
+    picture.src = projects[i].picture;
+    modalDescription.innerText = projects[i].description;
+    modalDemo.innerText = projects[i].demo;
+    modalSource.innerText = projects[i].source;
+  });
 };
```
Comment (moise10r) [inline js/dinamic.js:83]: - [ ] Excellent job of implementing the` see project` details button to show the popup with details about the project but I can see that it only works once at the first click. Kindly check your createModal function and make the `see project` button work  as expected and show project details every time the user clicks and not just once
1.

**Review by moise10r — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please make sure that the popup follows the design for both desktop and mobile versions as required, kindly change the margins, text-align,....

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/dinamic.js (hunk 1 above)
Comment (sergiogval) [reply js/dinamic.js:84]: Thanks for the observation. The bug was fixed.

**Review by moise10r — APPROVED** [review]
###  **Hi @sergiogval  ,** 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

Cheers and Happy coding!👏👏👏


 #### Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**